versioned_docs/version-8.5/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -4,64 +4,106 @@ title: "Red Hat OpenShift"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Compatibility
+Red Hat OpenShift, a Kubernetes distribution maintained by [Red Hat](https://www.redhat.com/en/technologies/cloud-computing/openshift), provides options for both managed and on-premise hosting.
 
-We test against the following OpenShift versions and guarantee compatibility with:
+Deploying Camunda 8 on Red Hat OpenShift is achievable using Helm, given the appropriate configurations. However, it's important to note that the Security Context Constraints (SCCs) and Routes configurations might require slight deviations from the guidelines provided in the [general Helm deployment guide](/self-managed/setup/install.md).
 
-| OpenShift version | Supported          |
-| ----------------- | ------------------ |
-| 4.12.x            | :white_check_mark: |
-| 4.13.x            | :white_check_mark: |
-| 4.14.x            | :white_check_mark: |
+## Cluster Specification
 
-Any version not explicitly marked in the table above is not tested, and we cannot guarantee compatibility.
+When deploying Camunda 8 on an OpenShift cluster, the cluster specification should align with your specific requirements and workload characteristics. Here's a suggested configuration to begin with:
 
-## Pitfalls to avoid
+- **Instance type:** 4 vCPUs (x86_64, >3.1 GHz), 16 GiB Memory (for example, [m5.xlarge on AWS](https://aws.amazon.com/en/ebs/general-purpose/))
+- **Number of dedicated nodes:** 4
+- **Volume type:** SSD volumes (with between 1000 and 3000 IOPS per volume, and a throughput of 1,000 MB/s per volume, for instance, [gp3 on AWS](https://aws.amazon.com/en/ebs/general-purpose/))
 
-For general deployment pitfalls, visit the [deployment troubleshooting guide](/self-managed/operational-guides/troubleshooting/troubleshooting.md).
+### Supported Versions
 
-### Security Context Constraints (SCCs)
+We conduct testing and ensure compatibility against the following OpenShift versions:
 
-Much like how roles control the permissions of users, Security Context Constraints (SCCs) are a way to control the permissions of the applications deployed, both at the pod and container level. It's generally recommended deploying your application with the most restricted SCCs possible. If you're not familiar with security context constraints, refer to the [OpenShift documentation](https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html).
+- Any version not listed below hasn't been tested, and we do not guarantee compatibility.
+- The OpenShift end of support date must be in the future: we do not ensure compatibility with versions no longer supported by Red Hat, as indicated in the "End of Support Date" column.
```

Comment:
Could we move lines 26 and 27 beneath the table, so the listed versions immediately follow the testing-and-compatibility confirmation (line 24)? If we'd like to draw attention to those requirements before the table, I might change it to a statement, for example `OpenShift versions for which we ensure compatibility are provided in the following table.`

Clearer language might look something like:

```suggestion
- Compatibility is not guaranteed for any OpenShift version not explicitly listed in the table.
- Compatibility is not guaranteed for OpenShift versions no longer supported by Red Hat, as indicated in the "End of Support Date" column.
```